index.js

```diff
@@ -123,7 +134,7 @@ function handleResults(error, results) {
   }
 
   // Clear command-line before printing results
-  process.stdout.write('\033c')
+  process.stdout.write('\x1Bc');
```

Comment:
I'm hesitant to make this change. I wasn't very familiar with either, but my search led to a warning that `\x1Bc` may not work properly in all situations.

https://stackoverflow.com/a/47503782/2020476

Were you seeing an issue with `\033c`?

Reply:
It works fine. But I had an eslint warning. But you're right, maybe it does not work correctly in all situations. I'll revert the change